aptos-move/block-executor/src/counters.rs

```diff
@@ -211,6 +213,22 @@ pub static BLOCK_COMMITTED_TXNS: Lazy<HistogramVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub static BLOCK_VIEW_DISTINCT_KEYS: Lazy<HistogramVec> = Lazy::new(|| {
+    register_avg_counter_vec(
+        "aptos_execution_block_view_distinct_keys",
```

Comment:
What "keys" does this refer to? Comments would be helpful.

Reply:
keys in the view, every map (resource/resource_group/modules/delayed_fields has different key), and each is broken down separately

I am going to leave documentation improvements to follow-up PR on main, as this is a cherry-pick to release branch, not the original PR